services/agent/src/simcore_service_agent/core/settings.py

```diff
@@ -11,7 +11,7 @@
 
 
 class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
-    LOGLEVEL: LogLevel = Field(
+    LOG_LEVEL: LogLevel = Field(
```

Comment:
is there not already this in BaseapplicationSettings in settings library?

Reply:
No it is not defined. We have some utilities inside `MixinLoggingSettings` for validating it. but the name is not enforced

Reply:
sad